src/state/slices/opportunitiesSlice/constants.ts

```diff
@@ -71,6 +76,13 @@ export const baseEarnFarmingOpportunity = {
   type: DefiType.Farming,
 }
 
+export const v5EarnFarmingOpportunity: EarnOpportunityType = {
+  ...baseEarnFarmingOpportunity,
+  assetId: foxEthStakingAssetIdV5,
+  contractAddress: fromAssetId(foxEthStakingAssetIdV5).assetReference,
+  opportunityName: 'Fox Farming V5',
```

Comment:
index this boy

Reply:
dc41ed574b8a8c21f7dc447d37fb638ad9de538b